docs/Release_Guide/release_steps/met/update_zenodo.rst

```diff
@@ -5,7 +5,24 @@ Update the DOI for this official release.
 
 .. dropdown:: Instructions
 
```

Comment:
Consider linking to the DTCenter Zenodo Community so we don't have to track it down when there are permission problems.
```suggestion

Official MET releases are published through `Zenodo <https://zenodo.org/>`_ and members of the `DTCenter Zenodo Community <https://zenodo.org/communities/dtcenter/members>`_ have permission to upload new software versions.

```